Makefile

```diff
@@ -0,0 +1,43 @@
+# Image names
+PROD_IMAGE_NAME=minutes-prod
+DEV_IMAGE_NAME=minutes-dev
+ORGANIZATION=ubclaunchpad
+TAG=latest
+
+# Ports
+PROD_PORT=80
+DEV_PORT=8080
+
+.PHONY: dev run build-prod build-dev push-dev push-prod
+
+all: build-prod build-dev
+
+run:
+	docker run -it \
+			-p $(PROD_PORT):$(PROD_PORT) \
+			$(PROD_IMAGE_NAME)
+
+dev:
+	echo "Opening dev environment on localhost port $(DEV_PORT)."
```

Comment:
`echo` should probably be `@echo` in order to suppress the echo command itself from being printed.